src/lifetime-elision.md

```diff
@@ -71,6 +71,14 @@ rules.
 If the trait object is used as a type argument of a generic type then the
 containing type is first used to try to infer a bound.
 
+* If there is a unique bound from the containing type then that is the default
+* If there is more than one bound from the containing type then an explicit
+  bound must be specified
+
+If the trait object is used as a binding for an associated type (e.g.,
```

Comment:
I'm not sure the "e.g." parenthetical is useful here. But if we keep it, it should say "for example". It'd be nice if "trait object" and "associated type" were links. Or link to the full example?

Reply:
Well, perhaps the e.g. isn't the most elegant way to express it, but I think that showing the syntax will go along way towards helping people understand what the other words actually mean. 

I can definitely make links.

Separate but relevant: In general, I tend to think we should deprecate the term trait objects and move towards "dyn types" or something like that, but that'd be a shift obviously. I do think though that leveraging the keyword in our terminology makes a lot of sense.